package.json

```diff
@@ -8,7 +8,7 @@
     "build": "yarn build:rollup && yarn build:next",
     "build:next": "cd ./site && next build && next export",
     "build:rollup": "rollup --config ./config/rollup/rollup.config.js",
-    "clean": "rimraf ./node_modules ./packages/*/{dist,lib,node_modules} ./site/{.next,out}",
+    "clean": "bash -c 'rimraf ./packages/*/{dist,lib,node_modules} ./site/{.next,out}'",
```

Comment:
As @cameracker reminded me, rimraf adds cross-platform support, and switch this to base breaks Windows users. Please restore this line. Thanks!